hapi/hapi/build.gradle.kts

```
@@ -3,7 +3,8 @@ plugins {
     id("org.hiero.gradle.module.library")
     id("org.hiero.gradle.feature.protobuf")
     id("org.hiero.gradle.feature.test-fixtures")
-    id("com.hedera.pbj.pbj-compiler") version "0.11.0"
+    id("com.hedera.pbj.pbj-compiler") version "0.11.3"
```

nit: If you come back
Or better yet leave as is with what you've tested but the main version of this cherry pick can use 0.11.6
```suggestion
    id("com.hedera.pbj.pbj-compiler") version "0.11.6"
```